src/main/java/com/okta/tools/helpers/SessionHelper.java

```diff
@@ -26,43 +26,41 @@ public SessionHelper(OktaAwsCliEnvironment environment, CookieHelper cookieHelpe
         this.cookieHelper = cookieHelper;
     }
 
-    /**
-     * Gets the current session file's path
-     *
-     * @return A {@link Path} for the current session file
-     * @throws IOException
-     */
-    private Path getSessionPath() throws IOException {
-        return FileHelper.resolveFilePath(FileHelper.getOktaDirectory(), ".current-session");
-    }
-
```

Comment:
:art: a good chunk of the diff in this file is method reordering to show methods in the order they are used (step down rule from Clean Code).